README.md

```diff
@@ -263,7 +263,7 @@ const networkMiddleware = createNetworkMiddleware();
 
 const store = createStore(
   rootReducer,
-  applyMiddleware([networkMiddleware, sagaMiddleware])
+  applyMiddleware(...[networkMiddleware, sagaMiddleware])
```

Comment:
Good catch! :)

Better to pass parameters directly without destructuring the array, which is less confusing:

`applyMiddleware(networkMiddleware, sagaMiddleware)`

Reply:
@adrienthiery are you planning to address this any time soon? It should be pretty quick.

Reply:
Hi @rauliyohmc ,

done. Sorry, I was away with no internet for the week.